config/routes.rb

```diff
@@ -1,7 +1,9 @@
 Rails.application.routes.draw do
 
   resources :todo_lists do
-    resources :todo_items
+    resources :todo_items do
+      get 'mark_done'
```

Comment:
I would have probably advocated for the `mark_done` call to be just an update call to the todo_item resource, but I understand why you went the way you did.